node/index.md

```diff
@@ -21,30 +21,62 @@ It's up to you to do whatever you want with this!
 {% highlight js %}
 var sassdoc = require('sassdoc');
 
-sassdoc.parse(__dirname + '/your-sass-folder').then(function (data) {
-  console.log(data);
-})
+sassdoc.parse(__dirname + '/your-sass-folder')
+  .then(function (data) {
+    console.log(data);
+  });
 {% endhighlight %}
 
 ## Generating documentation
 
-Or if you want to use SassDoc at its best and generate the whole documentation from Node, you can use the `default` function that returns a promise and accepts 2 arguments: the source folder, and a configuration object (which is optional).
+Or if you want to use SassDoc at its best and generate the whole documentation from Node, you can use the default function.
 
 {% highlight js %}
-// Require SassDoc main function
-var sassdoc = require('sassdoc').default;
-
-// Generate docs
-// @param {String} source - path to source or glob string/array
-// @param {Object} configuration - configuration object
-// @return {Promise}
-sassdoc(source, configuration).then(function () {
-  console.log('Your documentation has been generated!');
-}, function (err) {
-  console.error(err);
-});
+var sassdoc = require('sassdoc');
+
+sassdoc(source)
+  .then(function () {
+    console.log('Your documentation has been generated!');
+  }, function (err) {
+    console.error(err);
+  });
+{% endhighlight %}
+
+If you want to [configure][configuration] the destination directory, you can pass a custom `dest` in the configuration object.
+
+## Synopsis
+
+Both `sassdoc` and `sassdoc.parse` functions have the same synopsis:
+
+* `@param {String} source` path to source or [glob string/array](#about-glob);
+* `@param {Object} [configuration]` optional [configuration] object;
+* `@return {Promise}`.
+
+## Streaming
+
+I lied to you in the [Synopsis](#synopsis) part. There is one more way you can use these functions; if you ommit the `source` argument, you'll get a [transform stream][repo_through2], able to handle [vinyl files][repo_vinyl]. You can easily use it with [vinyl-fs][repo_vinyl_fs].
```

Comment:
Nowhere else in the docs do we speak using *I*. Don't use *I*. Use *we* at worst, no pronoun at best.